Makefile

```diff
@@ -7,7 +7,7 @@ BOND_DENOM = stake
 localnet-alliance-rmi:
 	docker rmi terra-money/localnet-alliance 2>/dev/null; true
 
-localnet-build-env: localnet-alliance-rmi
```

Comment:
Has this been commited by mistake? Because the old image should be always deleted